internal/cfgmgr/cfgmgr.go

```diff
@@ -0,0 +1,165 @@
+package cfgmgr
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"unicode/utf16"
+
+	winio "github.com/Microsoft/go-winio/pkg/guid"
+	errorspkg "github.com/pkg/errors"
+)
+
+//go:generate go run ../../mksyscall_windows.go -output zsyscall_windows.go cfgmgr.go
+
+//sys cmGetDeviceIDListSize(pulLen *uint32, pszFilter *byte, uFlags uint64) (hr error) = cfgmgr32.CM_Get_Device_ID_List_SizeA
+//sys cmGetDeviceIDList(pszFilter *byte, buffer *byte, bufferLen uint64, uFlags uint64) (hr error)= cfgmgr32.CM_Get_Device_ID_ListA
+//sys cmLocateDevNode(pdnDevInst *uint32, pDeviceID string, uFlags uint64) (hr error) = cfgmgr32.CM_Locate_DevNodeW
+//sys cmGetDevNodeProperty(dnDevInst uint32, propertyKey *devPropKey, propertyType *uint64, propertyBuffer *uint16, propertyBufferSize *uint64, uFlags uint64) (hr error) = cfgmgr32.CM_Get_DevNode_PropertyW
+
+type devPropKey struct {
+	fmtid winio.GUID
+	pid   uint64
+}
+
+const STRING_BUFFER_SIZE = uint64(1024)

```

Comment:
Is this a constant from the DLL? It's not used anywhere other than this package so why not strBufSize or similar?

Reply:
It's a constant used in the pnp code, but I don't think it's exported in a dll anywhere.